client/signup/steps/site-picker/test/site-picker-submit.jsx

```diff
@@ -74,8 +77,8 @@ describe( 'SitePickerSubmit', () => {
 
 	test( 'Does not blow up', () => {
```

Comment:
Does this test provide any real value? :P It seems very similar to some of the ones we removed in other PRs. Or maybe the title just needs an update?

Reply:
Yup, good call, the title needs an update - done in 01f10dbbee6f9c29615bccfdc69128cbe5d4b4b7.